tasks/copy-localization-config.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copy our lit-localize config to each package
+ */
+
+import fs from 'fs-extra';
+
+fs.readdirSync('./packages').forEach(pkg => {
+  fs.copySync(
+    './localization/lit-localize.json',
+    `./packages/${pkg}/lit-localize.json`
+  );
+});
```

Comment:
The following would be better for performance if we have a lot of projects later on (runs in parallel, but test it before commiting tho).

It could also be better to filter the list of packages (e.g. using lerna, like in other scripts, or by checking the package dependencies to see if it uses lit-localize), so the file is only copied to localized projects. But I can handle this later on, just open an issue so I don't forget to do it.

```suggestion
import process from 'node:process';

Promise.all(fs.readdirSync('./packages').map(async (pkg) => fs.copy(
    './localization/lit-localize.json',
    `./packages/${pkg}/lit-localize.json`
  );
})).then(console.log, (e) => {
    console.error(e);
    process.exitCode = 1;
});
```

Reply:
Changed to use promise.all in [7720088](https://github.com/blindnet-io/privacy-components-web/pull/118/commits/772008886835c404e5adbba73de4dbd621ce7a54)

Opened #120